code/code/sys/db.cc

```diff
@@ -1570,8 +1599,9 @@ bool zoneData::bootZone(int zone_nr) {
 
     if (rs.command == 'M' || rs.command == 'O' || rs.command == 'B' ||
         rs.command == 'C' || rs.command == 'K' || rs.command == 'E' ||
-        rs.command == 'P' || (rs.command == 'T' && !rs.if_flag) ||
-        rs.command == 'R' || rs.command == 'D' || rs.command == 'L')
+        rs.command == 'I' || rs.command == 'P' ||
+        (rs.command == 'T' && !rs.if_flag) || rs.command == 'R' ||
```

Comment:
minor nit: These parentheses are redundant (tho that won't hurt).
I think readability would be improved by moving the R case onto the line with P, and leaving the && case in its own line. In this case, () wouldn't be necessary anymore (though this is a minor improvement).

Reply:
I agree, but in interest of not forcing you to re-approve the PR I'm just going to go ahead and resolve this since it's just a aesthetic issue primarily, as I'd like to get this merged this morning. If I have to tweak anything in this code later on I'll make this change then.